<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Careers</title>

<script src="site_libs/header-attrs-2.19/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Jin group</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="people.html">People</a>
</li>
<li>
  <a href="careers.html">Careers</a>
</li>
<li>
  <a href="news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jinlab-washu">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://twitter.com/hashtag/peterjin">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/sheng-chih-jin-3b60a011">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Careers</h1>

</div>


<p><br></p>
<div id="postdoctoral-associate-in-functional-genomics"
class="section level3">
<h3>Postdoctoral Associate in Functional Genomics</h3>
<p>The Jin Lab is seeking a talented and enthusiastic postdoctoral
fellow to join our growing team in the Department of Genetics at
Washington University School of Medicine. To tackle central questions
about the pathophysiology of neuropathy, congenital hydrocephalus,
cerebral palsy, and/or congenital heart defects, the fellow will conduct
functional genomics research under the guiding aims:</p>
<ul>
<li>Characterization of biological mechanisms disrupted by pathogenic
mutations</li>
<li>Development of human disease models using massively parallel
reporter assays and/or zebrafish model systems</li>
<li>Implementation of multi-omics approach that integrate
high-throughput imaging and machine learning coupled with CRISPR/Cas9
screens</li>
</ul>
<p>Our interdisciplinary research environment brings together life
scientists, computational biologists, and physician-scientists to
identify mutations underlying human diseases and their pathogenic
mechanisms with the ultimate goal of translating these findings into
effective therapeutic treatments. For more information, please visit our
lab webpage: <a href="https://scjin.github.io/"
class="uri">https://scjin.github.io/</a></p>
<p>Required qualifications:</p>
<ul>
<li>Recent PhD/MD in a biological sciences or related fields (Molecular
Biology, Developmental Biology, Cell Biology, Neurosciences,
Biochemistry, etc.)</li>
<li>Experience in the zebrafish handling, cell culture, neuroscience,
and genomics</li>
<li>Good communication skills</li>
<li>In addition, applicants must be able to visit St. Louis for
potential interviews</li>
</ul>
<p>Applicants should email a single PDF file consisting of a letter of
interest and CV, and should arrange for three letters of reference to be
sent to Dr. Jin (<a href="mailto:jin810@wustl.edu"
class="email">jin810@wustl.edu</a>).</p>
<hr />
</div>
<div id="postdoctoral-associate-in-computational-genomics"
class="section level3">
<h3>Postdoctoral Associate in Computational Genomics</h3>
<p>The Jin lab is looking for an energetic and motivated individual to
work as a Postdoctoral Associate with a doctoral degree in genetics,
genomics, computational biology, or computer science to investigate the
genetic basis of neuropathy, congenital hydrocephalus, cerebral palsy,
and/or congenital heart defects.</p>
<p>Although advances in next-generation sequencing have discovered the
genetic causes of a large proportion of patients with congenital
disorders,the etiology of most cases remains unclear. It is becoming
evident that simple genetic models do not entirely explain the causes of
all congenital disorders and more nuanced approaches are required.
Therefore, the Jin lab’s research is focusing on the formulation and
development of computational methods to determine the complex genetic
basis underlying congenital disorders using human genetics, genomics,
and advanced statistical methods. Our goal is to translate an improved
molecular understanding of congenital disorders from the laboratory into
clinically actionable models, diagnostics, and ultimately therapeutic
interventions.</p>
<p>We are currently looking for researchers with a strong computational
background in genomic and functional genomic analysis, including whole
genome sequencing, single-cell RNA-sequencing, genetic association
studies, and integrative analysis of genomic and functional genomic
data. The candidate will be expected to plan and carry out research
tasks independently and write-up/present findings on a regular
basis.</p>
<p>The Jin lab is located in the Couch Biomedical Research building on
the 5th floor in the Department of Genetics space. The successful
candidate will take advantage of the unique collaborative relationships
with the McDonnell Genome Institute, the Intellectual and Developmental
Disabilities Research Center, Departments of Pediatrics &amp;
Neuroscience, and across the broader research community of Washington
University which provides outstanding training and educational
opportunities. Dr. Jin is committed to mentoring and training the
candidate, providing all the support the candidate needs to reach their
career goals. For competitive candidates, references from current and
previous trainees attesting to the supportive and dynamic research
environment of the Jin lab are available upon request.</p>
<p>To be considered for this role, please email Dr. Jin at <a
href="mailto:jin810@wustl.edu" class="email">jin810@wustl.edu</a> with
the following items: 1) CV with a list of publications; 2) a short
summary of your present and future research interests; 3) a list of
three references with their contact information. Diverse expertise will
be considered. Competitive salary and benefits will be provided</p>
<hr />
</div>
<div id="graduates-undergraduates-summer-interns-and-visiting-scholars"
class="section level3">
<h3>Graduates, Undergraduates, Summer Interns and Visiting Scholars</h3>
<p>If you are a graduate/undergraduate student or a visiting scholar and
looking for temporary lodging, please email your CV to Dr. Jin at <a
href="mailto:jin810@wustl.edu" class="email">jin810@wustl.edu</a> for
discussion.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
