<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Jin group</title>

<script src="site_libs/header-attrs-2.21/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Jin group</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="people.html">People</a>
</li>
<li>
  <a href="careers.html">Careers</a>
</li>
<li>
  <a href="news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jinlab-washu">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://twitter.com/hashtag/peterjin">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/sheng-chih-jin-3b60a011">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Jin group</h1>

</div>


<p><link rel="stylesheet" href="styles.css" type="text/css"></p>
<p><br></p>
<p>Welcome to the Jin group in the <a
href="http://genetics.wustl.edu/">Department of Genetics</a> and the <a
href="https://www.genome.wustl.edu/">McDonnell Genome Institute</a> at
the Washington University School of Medicine. Our mission is to provide
meaningful and interpretable insight into disease biology, and define
new targets for risk determination, prevention, and therapy. We are
currently focusing on the formation, development, and application of
human genetic, functional genomic, and bioinformatic methods to better
analyze and integrate genome sequencing, single-cell RNA-sequencing,
epigenomic, spatial genomic, and proteomic data. Through integration of
diverse type of omics data and epigenetic functional annotations, the
integrative genomic analysis will provide a better understanding of the
molecular basis of cardiovascular diseases and neurological disorders.
Following integrative genomic analyses, we use zebrafish and massively
parallel reporter assays to precisely model human mutations.</p>
<p>We collaborate with clinicians, the <a
href="https://thepnrr.org/">Peripheral Neuropathy Research Registry</a>,
the <a href="https://cprn.org/">Cerebral Palsy Research Network</a>, and
the <a href="https://sites.wustl.edu/undiagnosed/">WashU Undiagnosed
Diseases Network</a> to assemble thoroughly phenotyped cohorts for gene
discovery. We also collaborate with experimentalists to design scalable
high-throughput assays to model effects of disease-associated mutations.
If you think any of this sounds cool consider <a
href="./careers.html">joining us</a> in working to make the world a
better place.</p>
<p><img src="images/Jin_Lab_2023.png" style="width:100%; border:0; margin-right: 20px" align="top"></p>
<hr />
<p><img src="images/LabPoster_vDNA1024_1.jpg" style="width:100%; border:0; margin-right: 20px" align="top">
These posters are meant to show that racial justice and support for
marginalized communities cannot be separated from the practice of
science. We must actively work to recognize the obstacles that
scientists (and potential scientists) from marginalized communities
face, and dismantle structures of power that prevent them from
succeeding. We must also consider the effects of our research and
research choices on marginalized communities. Please visit <a
href="https://sammykatta.com/diversity">here</a> for more diversity and
inclusion lab posters!</p>
<hr />
<p>We are currently working on the following areas of research:</p>
<p><strong><em>Methods Development</em></strong></p>
<p>During my postdoctoral training, we developed a control-free
statistical approach and applied it to a whole-exome sequencing dataset
of 2,871 congenital heart disease probands procured from the <a
href="https://benchtobassinet.com/?page_id=36">Pediatric Cardiac
Genomics Consortium</a> to demonstrate that ~1.8% of cases are
attributed to rare inherited variants. First, we demonstrated that
extremely rare transmitted variants in each gene are similarly
distributed when compared to de novo mutations; this allowed for a
robust estimation of the expected frequency of rare transmitted variants
based on the expected frequency of de novo mutations. Second, analysis
of recessive variants has been confounded by varying degrees of
consanguinity among probands and differences in consanguinity between
cases and controls. We conceived and developed a novel method of
analysis using a simple polynomial model that accounts for this variable
inbreeding, enabling a powerful test comparing the observed and expected
recessive variants in each gene. Currently, I am working with Drs. <a
href="http://zhaocenter.org/">Hongyu Zhao</a> and <a
href="http://www.leklab.org/">Monkol Lek</a>, to develop novel methods
for large-scale genetic association analysis, joint modeling of de novo
and transmitted variants, and polygenic risk prediction.</p>
<p><strong><em>Unravelling Molecular Signatures and Novel Therapeutic
Targets for Idiopathic Peripheral Neuropathy</em></strong></p>
<p>Idiopathic peripheral neuropathy (IPN), a result of damage to the
peripheral nerves with unknown causes, is a debilitating, painful, and
as-yet-untreatable condition. Due to the unknown origin of the damage
and the lack of targeted therapy, treatment for patients with IPN
revolves around mere symptom management. In collaboration with Drs. <a
href="http://milbrandt.wustl.edu/jeff-milbrandt/">Jeffrey Milbrandt</a>,
<a
href="https://www.hopkinsmedicine.org/profiles/details/ahmet-hoke">Ahmet
Hoke</a>, <a href="https://sites.wustl.edu/diantonio/">Aaron Di
Antonio</a>, and the <a href="https://thepnrr.org/">Peripheral
Neuropathy Patient Registry</a>, we are integrating whole-genome
sequencing and SOMAscan proteomic data in extreme IPN patients with
negative clinical genetic screening results and performing bioinformatic
analysis to identify variants, pathways, and biomarkers for
pathophysiological clues to axon degeneration.</p>
<p><strong><em>Genomic Research of Cerebral Palsy and
Dystonia</em></strong></p>
<p>In collaboration with Drs. <a
href="https://www.kruerlab.org/">Michael Kruer</a>, <a
href="https://cruchagalab.wustl.edu/">Carlos Cruchaga</a>, the <a
href="https://dc.rarediseasesnetwork.org/">Dystonia Coalition</a>, and
the <a href="https://cprn.org/">Cerebral Palsy Research Network</a>, we
are applying an integrative, multidimensional omics approach to a large,
well-phenotyped cerebral palsy/dystonia cohort to catalyze gene
discovery, provide mechanistic insights into newly identified genetic
causes, make genotype-phenotype correlations, and create a metadata
ecosystem freely available to the cerebral palsy/dystonia community.</p>
<p><strong><em>Human Genetics and Molecular Mechanisms of Human
Neurological Diseases</em></strong></p>
<p>In collaboration with Drs. <a
href="https://www.massgeneral.org/doctors/22473/kristopher-kahle">Kristopher
Kahle</a>, <a href="https://sites.wustl.edu/krolllab/">Kristen
Kroll</a>, <a href="http://genetics.wustl.edu/jdlab/">Joe Dougherty</a>,
and the <a href="https://undiagnoseddiseases.wustl.edu/">WashU
Undiagnosed Diseases Network</a>, we are focusing on studying the
genetic underpinnings of rare Mendelian forms of human neurological
diseases, using whole-genome sequencing, quantitative phenotyping,
transcriptomic profiling, massively parallel reporter assays, and in
vitro/in vivo models.</p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
