---
title: "Jin group"
output: html_document
---

<link rel="stylesheet" href="styles.css" type="text/css">

<br> 

Welcome to the Jin group in the [Department of Genetics](http://genetics.wustl.edu/) and the [McDonnell Genome Institute](https://www.genome.wustl.edu/) at the Washington University School of Medicine. Our mission is to provide meaningful and interpretable insight into disease biology, and define new targets for risk determination, prevention, and therapy. We are currently focusing on the formation, development, and application of genetic, genomic, and bioinformatic methods to better analyze and integrate exome and genome sequencing, SNP array, RNA-sequencing, epigenomic, metabolomic, and proteomic data. Through integration of diverse type of transcriptomic and epigenetic functional annotations, the integrative genomic analysis will provide a better understanding of the molecular basis of cardiovascular diseases and neurodevelopmental disorders. Following integrative genomic analyses, we use zebrafish and massively parallel reporter assays to precisely model human mutations. 

We collaborate with clinicians, the [Pediatric Cardiac Genomics Consortium](https://benchtobassinet.com/?page_id=36), the [International Cerebral Palsy Genomics Consortium](https://icpgc.org/),the [WashU Undiagnosed Diseases Network](https://sites.wustl.edu/undiagnosed/), and the [Yale Center for Mendelian Genomics](https://medicine.yale.edu/keck/ycga/) to assemble thoroughly phenotyped cohorts for gene discovery. We also collaborate with experimentalists to design scalable high-throughput assays to model effects of disease-associated mutations. If you think any of this sounds cool consider [joining us](./careers.html) in working to make the world a better place.


<img src="images/Jin_Sheng_Chih_2022.png" style="width:100%; border:0; margin-right: 20px" align="top">


-----

 We are currently working on the following areas of research:

***Methods Development***

During my postdoctoral training, we developed a control-free statistical approach and applied it to a whole exome sequencing dataset of 2,871 CHD probands procured from the [Pediatric Cardiac Genomics Consortium](https://benchtobassinet.com/?page_id=36) to demonstrate that ~1.8% of cases are attributed to rare inherited variants. First, we demonstrated that extremely rare transmitted variants in each gene are similarly distributed when compared to de novo mutations; this allowed for a robust estimation of the expected frequency of rare transmitted variants based on the expected frequency of de novo mutations. Second, analysis of recessive variants has been confounded by varying degrees of consanguinity among probands and differences in consanguinity between cases and controls. We conceived and developed a novel method of analysis using a simple polynomial model that accounts for this variable inbreeding, enabling a powerful test comparing the observed and expected recessive variants in each gene. Currently, I am working with Drs. [Qiongshi Lu](https://qlu-lab.org/), [Hongyu Zhao](http://zhaocenter.org/), and [James Knight](https://medicine.yale.edu/profile/j_knight/), to develop novel methods for large-scale genetic association analysis, joint modeling of de novo and transmitted variants, and polygenic risk prediction.

***Complex Genetic Basis of Congenital Heart Disease***

Congenital heart disease affects ~1% of live births and remains the leading cause of mortality from birth defects. So far, despite extensive efforts by many groups, the etiology of ~55% of congenital heart disease cases still cannot be explained by environmental triggers, chromosomal abnormalities, or monogenic etiologies. It is becoming evident that simple genetic models do not entirely explain the causes of all congenital heart disease cases. More nuanced approaches are required; in collaboration with Drs. [Richard Lifton](https://www.rockefeller.edu/our-scientists/heads-of-laboratories/1139-richard-p-lifton/), [Martina Brueckner](https://medicine.yale.edu/lab/brueckner/), [Monkol Lek](http://www.leklab.org/), and [Peter Gruber](https://medicine.yale.edu/profile/peter_gruber/), we aim to develop novel computational methods to determine the complex genetic basis of congenital heart disease, with a specific focus on the interplay between the common non-coding and rare protein-coding variants by jointly analyzing whole exome/genome sequencing and SNP array data from the [Pediatric Cardiac Genomics Consortium](https://benchtobassinet.com/?page_id=36).

***Human Genetics and Molecular Mechanisms of Human Neurological Diseases*** 

In collaboration with Drs. [Kristopher Kahle](https://medicine.yale.edu/lab/kahle/) and [Richard Lifton](https://www.rockefeller.edu/our-scientists/heads-of-laboratories/1139-richard-p-lifton/), we are focusing on studying the genetic underpinnings of rare Mendelian forms of human neurological diseases, including congenital hydrocephalus, Chiari malformation, Arachnoid cysts, and Moyamoya disease, for the [Yale Center for Mendelian Genomics](http://mendelian.org/).

***Genomic Research of Cerebral Palsy***

As the lead computational biologist for the [International Cerebral Palsy Genomics Consortium](https://icpgc.org/), we aim to work as an integrated, collaborative international network of clinicians, researchers, and advocates in order to unlock the molecular basis of cerebral palsy using genomics and related tools, and to facilitate the development of novel therapies to improve the health and quality of life of individuals with cerebral palsy.
