---
title: "Jin group"
output: html_document
---

<link rel="stylesheet" href="styles.css" type="text/css">

<br> 

Welcome to the Jin group in the [Department of Genetics](http://genetics.wustl.edu/) and the [McDonnell Genome Institute](https://www.genome.wustl.edu/) at the Washington University School of Medicine. Our mission is to provide meaningful and interpretable insight into disease biology, and define new targets for risk determination, prevention, and therapy. We are currently focusing on the formation, development, and application of genetic, genomic, and bioinformatic methods to better analyze and integrate exome and genome sequencing, SNP array, RNA-sequencing, epigenomic, metabolomic, and proteomic data. Through integration of diverse type of transcriptomic and epigenetic functional annotations, the integrative genomic analysis will provide a better understanding of the molecular basis of cardiovascular diseases and neurodevelopmental disorders. Following integrative genomic analyses, we use zebrafish and massively parallel reporter assays to precisely model human mutations. 

We collaborate with clinicians, the [Peripheral Neuropathy Research Registry](https://thepnrr.org/), the [Cerebral Palsy Research Network](https://cprn.org/), the [WashU Undiagnosed Diseases Network](https://sites.wustl.edu/undiagnosed/), and the [Yale Center for Mendelian Genomics](https://medicine.yale.edu/keck/ycga/) to assemble thoroughly phenotyped cohorts for gene discovery. We also collaborate with experimentalists to design scalable high-throughput assays to model effects of disease-associated mutations. If you think any of this sounds cool consider [joining us](./careers.html) in working to make the world a better place.


<img src="images/Jin_Sheng_Chih_2022.png" style="width:100%; border:0; margin-right: 20px" align="top">


-----

 We are currently working on the following areas of research:

***Methods Development***

During my postdoctoral training, we developed a control-free statistical approach and applied it to a whole-exome sequencing dataset of 2,871 congenital heart disease probands procured from the [Pediatric Cardiac Genomics Consortium](https://benchtobassinet.com/?page_id=36) to demonstrate that ~1.8% of cases are attributed to rare inherited variants. First, we demonstrated that extremely rare transmitted variants in each gene are similarly distributed when compared to de novo mutations; this allowed for a robust estimation of the expected frequency of rare transmitted variants based on the expected frequency of de novo mutations. Second, analysis of recessive variants has been confounded by varying degrees of consanguinity among probands and differences in consanguinity between cases and controls. We conceived and developed a novel method of analysis using a simple polynomial model that accounts for this variable inbreeding, enabling a powerful test comparing the observed and expected recessive variants in each gene. Currently, I am working with Drs. [Qiongshi Lu](https://qlu-lab.org/), [Hongyu Zhao](http://zhaocenter.org/), and [Monkol Lek](http://www.leklab.org/), to develop novel methods for large-scale genetic association analysis, joint modeling of de novo and transmitted variants, and polygenic risk prediction.

***Unravelling Molecular Signatures and Novel Therapeutic Targets for Idiopathic Peripheral Neuropathy***

Idiopathic peripheral neuropathy (IPN), a result of damage to the peripheral nerves with unknown causes, is a debilitating, painful, and as-yet-untreatable condition. Due to the unknown origin of the damage and the lack of targeted therapy, treatment for patients with IPN revolves around mere symptom management. In collaboration with Drs. [Jeffrey Milbrandt](http://milbrandt.wustl.edu/jeff-milbrandt/), [Ahmet Hoke](https://www.hopkinsmedicine.org/profiles/details/ahmet-hoke), [Aaron Di Antonio](https://sites.wustl.edu/diantonio/), and the [Peripheral Neuropathy Patient Registry](https://thepnrr.org/), we are integrating whole-genome sequencing and SOMAscan proteomic data in extreme IPN patients with negative clinical genetic screening results and performing bioinformatic analysis to identify variants, pathways, and biomarkers for pathophysiological clues to axon degeneration.

***Human Genetics and Molecular Mechanisms of Human Neurological Diseases*** 

In collaboration with Drs. [Kristopher Kahle](https://medicine.yale.edu/lab/kahle/), [Kristen Kroll](https://sites.wustl.edu/krolllab/research/), [Joe Dougherty](http://genetics.wustl.edu/jdlab/), and the [WashU Undiagnosed Diseases Network](https://sites.wustl.edu/undiagnosed/), we are focusing on studying the genetic underpinnings of rare Mendelian forms of human neurological diseases, including congenital hydrocephalus, Vein of Galen malformation, Arachnoid cysts, and Moyamoya disease, using whole-genome sequencing, quantitative phenotyping, transcriptomic profiling, massively parallel reporter assyas, and in vitro/in vivo models.

***Genomic Research of Cerebral Palsy***

In collaboration with Drs. [Michael Kruer](https://www.kruerlab.org/), [Toni Pearson](https://physicians.wustl.edu/people/toni-pearson-md/), [Bhooma Aravamuthan](https://aravamuthanlab.wustl.edu/), [Chris Gurnett](https://gurnettlab.wustl.edu/), [Joe Dougherty](http://genetics.wustl.edu/jdlab/), and the [Cerebral Palsy Research Network](https://cprn.org/), we are applying an integrative, multidimensional omics approach to a large, well-phenotyped cerebral palsy cohort to catalyze gene discovery, provide mechanistic insights into newly identified genetic causes, make genotype-phenotype correlations, and create a metadata ecosystem freely available to the cerebral palsy community.
